app/src/ui/changes/filter-changes-list.tsx

```diff
@@ -1456,35 +1661,93 @@ export class FilterChangesList extends React.Component<
   private getNoResultsMessage = () => {
     if (
       this.props.filterText === '' &&
-      !this.props.includedChangesInCommitFilter
+      !this.props.includedChangesInCommitFilter &&
+      !this.props.filterNewFiles &&
+      !this.props.filterModifiedFiles &&
+      !this.props.filterDeletedFiles &&
+      !this.props.filterExcludedFiles
     ) {
       return undefined
     }
 
-    const filterTextMessage = this.props.filterText
-      ? ` matching your filter of '${this.props.filterText}'`
-      : ''
+    const activeFilters: string[] = []
+
+    if (this.props.filterText) {
+      activeFilters.push(`"${this.props.filterText}"`)
+    }
+
+    if (this.props.includedChangesInCommitFilter) {
+      activeFilters.push('Included in commit')
+    }
+
+    if (this.props.filterExcludedFiles) {
+      activeFilters.push('Excluded from commit')
+    }
+
+    if (this.props.filterNewFiles) {
+      activeFilters.push('New files')
+    }
+
+    if (this.props.filterModifiedFiles) {
+      activeFilters.push('Modified files')
+    }
 
-    const includedCommitText = this.props.includedChangesInCommitFilter
-      ? ' that are to be included in your commit'
-      : ''
+    if (this.props.filterDeletedFiles) {
+      activeFilters.push('Deleted files')
+    }
 
-    const conjunction = filterTextMessage && includedCommitText ? ' and ' : ''
+    if (activeFilters.length === 0) {
+      return undefined
+    }
 
-    return `Sorry, I can't find any changed files${filterTextMessage}${conjunction}
-        ${includedCommitText}.`
+    const bulletedList = activeFilters.map(filter => `• ${filter}`).join('\n')
+    return `Sorry, I can't find any changed files matching the following filters:\n${bulletedList}`
```

Comment:
![Image](https://github.com/user-attachments/assets/0297194c-d969-497e-8883-1a45b3ee7a25)

I know I recommended the bullet points (I wasn't thinking about it being centered). But.. I think we need another approach for our more visual users as centered aligned bullets feel a bit rough. 

I am torn on a recommendation tho, I believe the bullet list does help make clear what is impacting the no results message, but likely unless we have a visually "not part of a sentence" UI here, then they need to be comma delimited.

Reply:
I'll switch from bullet points to a comma-delimited list, which will look cleaner while maintaining readability.